README.md

```diff
@@ -219,35 +203,67 @@ Mutation output is grouped by selection groups. Each group contains three sectio
     end
    -----------------------
    ```
+   
+Concurrency
+-----------
 
-Presentations
--------------
+By default, mutant will test mutations in parallel by running up to one process for each core on your system. You can control the number of processes created using the --jobs argument.
```

Comment:
Can we quote the jobs flag like `--jobs` this will result in better readability.